### PR TITLE
feature/LogBox

### PR DESCRIPTION
**Pull request by idkjs**
Closes no issues.

I got tired of seeing the warning so I figured we needed a binding to it.

Best!

### CODE REVIEW

File: src/apis/LogBox.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+id: apis/LogBox
+title: LogBox
+officialDoc: https://reactnative.dev/docs/debugging#logbox
+---
+## [LogBox](https://reactnative.dev/docs/debugging#logbox)
```
Comment (MoOx) [inline src/apis/LogBox.md:6]: This title is redundant, page will generate it from `title`.
Also you can then update the `###` to `##`

File: src/apis/LogBox.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Console errors and warnings are displayed as on-screen notifications with a red or yellow badge, and the number of errors or warning in the console respectively. To view a console error or warnings, tap the notification to view the full screen information about the log and to paginiate through all of the logs in the console.
+
+> type signature
+
+```reason
+ignoreLogs: array(string) => unit
+```
+> example
+```reason
```
Comment (MoOx) [inline src/apis/LogBox.md:23]:
```suggestion
```

> example

```reason
```

File: src/apis/LogBox.md (hunk 1 above)
Comment (idkjs) [reply src/apis/LogBox.md:6]: Removed it.

Comment (MoOx) [reply src/apis/LogBox.md:6]: I still see this. And I don't see the other title adjusted neither.

Comment (idkjs) [reply src/apis/LogBox.md]: Must not have pushed the edit. I see it here now.

**Comment by MoOx** [comment]
I pushed a minor change. Merging! Thanks!

**Review by MoOx — APPROVED** [review]

**Pull request merged**